akka-stream/src/main/scala/akka/stream/javadsl/Flow.scala

```diff
@@ -991,6 +991,7 @@ final class Flow[-In, +Out, +Mat](delegate: scaladsl.Flow[In, Out, Mat]) extends
    * This operator makes it possible to extend the `Flow` API when there is no specialized
    * operator that performs the transformation.
    */
+  @deprecated("As Stage is deprecated.", "2.4.2")
```

Comment:
And, I think the message should guide the user about what to do instead.>

Is it FlowOps.via?